daemon/vpn/nordlynx/libtelio/libtelio_test.go

```diff
@@ -137,69 +129,70 @@ func Test_TelioDefaultConfig(t *testing.T) {
 func Test_TelioConfig(t *testing.T) {
 	category.Set(t, category.Integration)
 
-	expectedCfg := telioRemoteTestConfig
+	expectedCfg := toTelioFeatures(t, telioRemoteTestConfig)
 
 	remoteConfigGetter := mockVersionGetter{telioRemoteTestConfig}
 
-	cfg, err := handleTelioConfig(exampleEventPath, exampleDeviceID, exampleAppVersion, true, &remoteConfigGetter)
+	actualCfg, err := handleTelioConfig(exampleEventPath, exampleDeviceID, exampleAppVersion, true, &remoteConfigGetter)
 
 	assert.NoError(t, err)
 
-	var j1, j2 telioFeatures
-	err1 := json.Unmarshal([]byte(expectedCfg), &j1)
-	err2 := json.Unmarshal(cfg, &j2)
-
-	assert.NoError(t, err1)
-	assert.NoError(t, err2)
+	if diff := cmp.Diff(actualCfg, &expectedCfg, cmpopts.SortSlices(func(a, b teliogo.EndpointProvider) bool {
+		return a < b
+	})); diff != "" {
+		t.Errorf("Telio Config mismatch (-want +got):\n%s", diff)
+	}
+}
 
-	assert.True(t, reflect.DeepEqual(j1, j2))
+func toTelioFeatures(t *testing.T, cfg string) teliogo.Features {
+	features, err := teliogo.DeserializeFeatureConfig(cfg)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return features
 }
 
 func Test_TelioConfigLanaDisabled(t *testing.T) {
 	category.Set(t, category.Integration)
 
-	expectedCfg := telioRemoteTestConfigLanaDisabled
-
 	remoteConfigGetter := mockVersionGetter{telioRemoteTestConfigLanaDisabled}
 
 	cfg, err := handleTelioConfig(exampleEventPath, exampleDeviceID, exampleAppVersion, true, &remoteConfigGetter)
 
 	assert.NoError(t, err)
 
-	var j1, j2 telioFeatures
-	err1 := json.Unmarshal([]byte(expectedCfg), &j1)
-	err2 := json.Unmarshal(cfg, &j2)
-
-	assert.NoError(t, err1)
-	assert.NoError(t, err2)
-
-	assert.Nil(t, j2.Lana)
-	assert.Nil(t, j2.Nurse)
+	assert.Nil(t, cfg.Lana)
+	assert.Nil(t, cfg.Nurse)
 }
 
 func Test_TelioConfigAllDisabled(t *testing.T) {
 	category.Set(t, category.Integration)
 
-	expectedCfg := telioRemoteTestConfigAllDisabled
-
 	remoteConfigGetter := mockVersionGetter{telioRemoteTestConfigAllDisabled}
 
 	cfg, err := handleTelioConfig(exampleEventPath, exampleDeviceID, exampleAppVersion, true, &remoteConfigGetter)
 
 	assert.NoError(t, err)
 
-	var j1, j2 telioFeatures
-	err1 := json.Unmarshal([]byte(expectedCfg), &j1)
-	err2 := json.Unmarshal(cfg, &j2)
+	assert.Nil(t, cfg.Lana)
+	assert.Nil(t, cfg.Nurse)
+	assert.Nil(t, cfg.Derp)
+	assert.Nil(t, cfg.Direct)
+
+	// defaults from libtelio
+	assert.NotNil(t, cfg.Wireguard)
+	assert.NotNil(t, cfg.Wireguard.PersistentKeepalive)
+
+	assert.NotNil(t, cfg.Wireguard.PersistentKeepalive.Vpn)
+	assert.Equal(t, uint32(25), *cfg.Wireguard.PersistentKeepalive.Vpn)
 
-	assert.NoError(t, err1)
-	assert.NoError(t, err2)
+	assert.Equal(t, uint32(5), cfg.Wireguard.PersistentKeepalive.Direct)
 
-	assert.Nil(t, j2.Lana)
-	assert.Nil(t, j2.Nurse)
-	assert.Nil(t, j2.Derp)
-	assert.Nil(t, j2.Direct)
-	assert.Nil(t, j2.Wireguard)
+	assert.NotNil(t, cfg.Wireguard.PersistentKeepalive.Proxying)
+	assert.Equal(t, uint32(25), *cfg.Wireguard.PersistentKeepalive.Proxying)
```

Comment:
Plz give meaning to `magic values` by defining named constants

Reply:
Done